UI/window-basic-main-outputs.cpp

```diff
@@ -1290,17 +1363,16 @@ bool SimpleOutput::ReplayBufferActive() const
 struct AdvancedOutput : BasicOutputHandler {
 	OBSEncoder streamAudioEnc;
 	OBSEncoder streamArchiveEnc;
-	OBSEncoder aacTrack[MAX_AUDIO_MIXES];
+	OBSEncoder recordTrack[MAX_AUDIO_MIXES];
 	OBSEncoder videoStreaming;
 	OBSEncoder videoRecording;
 
 	bool ffmpegOutput;
 	bool ffmpegRecording;
 	bool useStreamEncoder;
+	bool useStreamAudioEncoder;
```

Comment:
Not sure how much sense it really makes to have this option. For video encoding it's to save resources, but in the case of audio you're still creating separate encoders for streaming and recording. And by making it the default you're likely breaking existing setups for streaming FTL + recording locally since it would now use Opus wouldn't it?

Reply:
> Not sure how much sense it really makes to have this option. For video encoding it's to save resources, but in the case of audio you're still creating separate encoders for streaming and recording.

The point of this is just to allow the user to indicate to OBS to use the same codec/encoder type as the stream output, if the user change the streaming one the recording change too. This is not a question of saving resources.

> And by making it the default you're likely breaking existing setups for streaming FTL + recording locally since it would now use Opus wouldn't it?

I should maybe add a piece of code to make those user use AAC on recording if the config string does not exist in the ini file.

Reply:
I think the default should just remain AAC. There really is no reason for it to follow the stream output by default since that has the potential of breaking setups when switching to a service that uses Opus, which would be unexpected and not visible in the UI until starting the recording fails.